components/pill/__docs__/storybook-stories.jsx

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import IconSettings from '~/components/icon-settings';
+import { PILL } from '~/utilities/constants';
+import Pill from '~/components/pill';
+import Icon from '~/components/icon';
+import Avatar from '~/components/avatar';
+
+function noop () {
```

Comment:
thanks for pointing. will add.

Reply:
`console.log` are ok, too, if needed. What I've done in the past, is pass `action` into the example as a prop and call it from there, since `action` won't exist in the doc site.